sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -480,62 +485,165 @@ def query_items(
+            kwargs['priority'] = kwargs['priority']
         feed_options = _build_options(kwargs)
-        feed_options["isStartFromBeginning"] = is_start_from_beginning
-        if start_time is not None and is_start_from_beginning is False:
-            feed_options["startTime"] = start_time.astimezone(timezone.utc).strftime('%a, %d %b %Y %H:%M:%S GMT')
-        if partition_key_range_id is not None:
-            feed_options["partitionKeyRangeId"] = partition_key_range_id
-        if partition_key is not None:
-            feed_options["partitionKey"] = self._set_partition_key(partition_key)
-        if max_item_count is not None:
-            feed_options["maxItemCount"] = max_item_count
-        if continuation is not None:
-            feed_options["continuation"] = continuation
+
+        change_feed_state_context = {}
+        # Back compatibility with deprecation warnings for partition_key_range_id
+        if (args and args[0] is not None) or is_key_exists_and_not_none(kwargs, "partition_key_range_id"):
+            warnings.warn(
+                "partition_key_range_id is deprecated. Please pass in feed_range instead.",
+                DeprecationWarning
+            )
+
+            try:
+                change_feed_state_context["partitionKeyRangeId"] = kwargs.pop('partition_key_range_id')
+            except KeyError:
+                change_feed_state_context['partitionKeyRangeId'] = args[0]
+
+        # Back compatibility with deprecation warnings for is_start_from_beginning
+        if (len(args) >= 2 and args[1] is not None) or is_key_exists_and_not_none(kwargs, "is_start_from_beginning"):
+            warnings.warn(
+                "is_start_from_beginning is deprecated. Please pass in start_time instead.",
+                DeprecationWarning
+            )
+
+            try:
+                is_start_from_beginning = kwargs.pop('is_start_from_beginning')
+            except KeyError:
+                is_start_from_beginning = args[1]
+
+            if is_start_from_beginning:
+                change_feed_state_context["startTime"] = "Beginning"
+
+        # parse start_time
+        if is_key_exists_and_not_none(kwargs, "start_time"):
+            if change_feed_state_context.get("startTime") is not None:
+                raise ValueError("is_start_from_beginning and start_time are exclusive, please only set one of them")
+
+            start_time = kwargs.pop('start_time')
+            if not isinstance(start_time, (datetime, str)):
+                raise TypeError(
+                    "'start_time' must be either a datetime object, or either the values 'now' or 'beginning'.")
+            change_feed_state_context["startTime"] = start_time
+
+        # parse continuation token
+        if len(args) >= 3 and args[2] is not None or is_key_exists_and_not_none(feed_options, "continuation"):
+            try:
+                continuation = feed_options.pop('continuation')
```

Comment:
no - currently if the continuation token being provided, then it will ignore the other parameters like start_time, feed_range, partition_key_range_id and partition_key. 

Since there are two places the continuation token can be passed in, one from the public api, one when calling by_page, so the real check happens in `ChangeFeedIterable`